en/app.ftl

```diff
@@ -370,12 +370,19 @@ banner-download-install-extension-cta = Add { -brand-name-relay } to { -brand-na
 banner-upgrade-headline = Upgrade to { -brand-name-relay-premium }
 banner-upgrade-copy = { -brand-name-firefox } { -brand-name-relay-premium } makes creating email aliases even easier, with custom alias domains and unlimited aliases.
 banner-upgrade-cta = Upgrade to { -brand-name-relay-premium }
+banner-label-action = Action step
 banner-choose-subdomain-headline = Choose your own domain
 banner-choose-subdomain-headline-aliases = Get your own custom alias domain
+banner-choose-subdomain-headline-aliases-v2 = Get a custom domain for your aliases
```

Comment:
I feel like this line is very similar to existing strings. Is this worth adding a new string for? 
- "Get your own custom alias domain" (`banner-choose-subdomain-headline`)
- "You can choose a custom domain for your email aliases." (`banner-choose-subdomain-copy`)